pkg/authn/keychain.go

```diff
@@ -44,6 +44,10 @@ type Keychain interface {
 	Resolve(Resource) (Authenticator, error)
 }
 
+type KeychainMany interface {
```

Comment:
`MultiKeychain` is probably a better name, but then we have some conflict with the method `authn.NewMultiKeychain`, which does a different thing. 😕

Reply:
Haven't got better idea